app/src/main/java/com/rage/clamber/AsyncTasks/LoginAsyncTask.java

```diff
@@ -0,0 +1,88 @@
+package com.rage.clamber.AsyncTasks;
+
+import android.os.AsyncTask;
+import android.support.annotation.Nullable;
+import android.util.Log;
+
+import org.json.JSONException;
+import org.json.JSONObject;
+
+import java.io.BufferedReader;
+import java.io.IOException;
+import java.io.InputStreamReader;
+import java.net.HttpURLConnection;
+import java.net.URL;
+
+/**
+ * Created by rage on 3/18/16.
```

Comment:
Throw in a brief description of what this async task is meant to accomplish here